cmd/kyma/sync/sync.go

```diff
@@ -0,0 +1,19 @@
+package sync
+
+import (
+	"github.com/kyma-project/cli/cmd/kyma/sync/function"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+//NewCmd creates a new function command
+func NewCmd(o *cli.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "sync",
+		Short: "Synchronises the function files from the cluster.",
+		Long:  "Use this command to synchronise the function files from the cluster.",
```

Comment:
I think that `sync` command may be used by the other functionalities in the future. Maybe the description should be more generic?